src/app/containers/Include/index.jsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable react/no-danger */
+import React, { useEffect, useState } from 'react';
+import 'isomorphic-fetch';
+import { string } from 'prop-types';
+import { GridItemConstrainedMedium } from '#lib/styledGrid';
+import webLogger from '#lib/logger.web';
+import useToggle from '../Toggle/useToggle';
+
+const logger = webLogger();
+
+const IncludeContainer = ({ href }) => {
+  const [markup, setMarkup] = useState(null);
+  const [hasError, setError] = useState(false);
+  const { enabled } = useToggle('include');
+
+  useEffect(() => {
+    const fetchMarkup = async () => {
+      try {
+        const res = await fetch(href);
+        if (res.status !== 200) {
+          throw new Error('Failed to fetch');
+        } else {
+          const html = await res.text();
+          setMarkup(html);
+        }
+      } catch (e) {
+        setError(true);
+        logger.error(`HTTP Error: "${e}"`);
+      }
+    };
+    if (enabled) {
+      fetchMarkup();
+    }
+  }, [href, enabled]);
+
+  const shouldNotDisplayInclude = hasError || !(markup && enabled);
```

Comment:
```suggestion
  const shouldNotDisplayInclude = hasError || !markup || !enabled;
```
I struggled to follow this statement, my suggestion just represents:
Don't display the include if:
- We had an error fetching the include: `hasError`
- The markup is empty: `!markup`
- Includes not enabled: `!enabled`